src/components/login/LoginForm.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import useAccessToken from '../../hooks/useAccessToken';
+import useLoginFormStore from '../../hooks/useLoginFormStore';
+import Button from '../ui/Button';
+import ErrorMessage from '../ui/ErrorMessage';
+import TextBox from '../ui/TextBox';
+
+interface LoginForm {
+  onClickMoveSignUp: () => void;
+}
+
+export default function LoginForm({
+  onClickMoveSignUp,
+}: LoginForm) {
+  const { setAccessToken } = useAccessToken();
+
+  const [{
+    username, password, errorMessage, accessToken,
+  }, store] = useLoginFormStore();
+
+  useEffect(() => {
+    if (accessToken) {
+      setAccessToken(accessToken);
+    }
+  }, [accessToken]);
+
+  const handleChangeUsername = (value: string) => {
+    store.changeUsername(value);
+  };
+
+  const handleChangePassword = (value: string) => {
+    store.changePassword(value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    store.login();
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <TextBox
+          label="아이디"
+          value={username}
+          onChange={handleChangeUsername}
+        />
+        <TextBox
+          label="비밀번호"
+          value={password}
+          type="password"
+          onChange={handleChangePassword}
+        />
+        <Button
+          size="fit"
+          color="gray"
+          onClick={onClickMoveSignUp}
+          marginTop
+        >
+          회원가입
+        </Button>
+        <Button
+          marginTop
+          type="submit"
+        >
+          로그인
+        </Button>
+        {errorMessage.length > 0 && (
+          <ErrorMessage>{errorMessage}</ErrorMessage>
+        )}
```

Comment:
p5: 저라면 이렇게 작성했을 것 같기도 해요. 문자열이다보니 빈 문자열인 경우만 체크를 하면 될 것 같더라구요!

```suggestion
        {errorMessage && <ErrorMessage>{errorMessage}</ErrorMessage>}
```